google/cloud/forseti/services/scanner/dao.py

```diff
@@ -63,6 +63,7 @@ class Violation(base):
         full_name = Column(String(1024))
         inventory_data = Column(Text(16777215))
         inventory_index_id = Column(String(256))
+        invocation_id = Column(DateTime())
```

Comment:
This is not an `id`.  This should either be `invocation_time` or `invocated_at` to be consistent with what we are using elsewhere, and also to better rerflect the column's DateTime type.

Also, within the context of this table, there is no idea of what `invocation` is. I know that `scanner` may not be future-proof, but we need to find something more descriptive.  Perhaps one possibility is to call this `audit_time`.

Reply:
`audit_invocation_time` it is.